### ISSUE TITLE
[Dev Edition] Admin Top Bar experiment - "My Account" should open in new tab

### ISSUE THREAD

**Issue by d19dotca** [elementor/elementor#15276]
## Prerequisites
- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest development version of Elementor.

## Description
I enabled the new Admin Top Bar experiment in the latest Dev Edition build, and noticed on the Elementor Settings page, clicking on "My Account" in the top right of the top bar immediately takes me out of my website and into my.elementor.com. I'd like to propose that it only open in a new tab in order to prevent users from accidentally leaving their WordPress admin area.

<img width="242" alt="image" src="https://user-images.githubusercontent.com/8153796/121296351-59ff6180-c8a5-11eb-9dc7-83312219e23d.png">

## Steps to reproduce
1. Enable the Admin Top Bar experiment.
2. Click on "My Elementor" in top right of the new Admin Top Bar.
3. Notice that it takes the user away from WordPress & Elementor and into the Elementor website instead. This is what should be in a new tab instead to avoid bringing users out of the WordPress admin area.

## Isolating the problem
- [X] This bug happens with only Elementor plugin active (and Elementor Pro) with development versions.
- [X] This bug happens with a vanilla WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
					
== Server Environment ==
	Operating System: Linux
	Software: Apache/2.4.41 (Ubuntu)
	MySQL version: (Ubuntu) v8.0.23
	PHP Version: 7.4.3
	PHP Max Input Vars: 3000
	PHP Max Post Size: 256M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.7.2
	Site URL: <siteURL>
	Home URL: <siteURL>
	WP Multisite: No
	Max Upload Size: 256 MB
	Memory limit: 1024M
	Permalink Structure: /%category%/%postname%/
	Language: en-CA
	Timezone: America/Vancouver
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.3.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_CA
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.1.2 Safari/605.1.15

== Active Plugins ==
	AuthLDAP
		Version: 2.4.10
		Author: Andreas Heigl <andreas@heigl.org>

	Better Search Replace
		Version: 1.3.4
		Author: Delicious Brains

	Elementor
		Version: 3.3.0-dev15
		Author: Elementor.com

	Elementor Beta (Developer Edition)
		Version: 1.1.1
		Author: Elementor.com

	Elementor Pro
		Version: 3.4.0-dev13
		Author: Elementor.com

	MainWP Child
		Version: 4.1.6.1
		Author: MainWP

	Pods - Custom Content Types and Fields
		Version: 2.7.28
		Author: Pods Framework Team

	Redis Object Cache
		Version: 2.0.19-dev
		Author: Till Krüss

	SMTP Mailer
		Version: 1.1.2
		Author: naa986

	ToolKit For Elementor
		Version: 1.3.12
		Author: ToolKit For Elementor


== Elementor - Compatibility Tag ==
	
	Elementor Pro: Incompatible
	ToolKit For Elementor: Incompatible

== Elementor Pro - Compatibility Tag ==
	
	ToolKit For Elementor: Incompatible

== Elements Usage ==
	
	archive : 1
		archive-posts : 1
		theme-archive-title : 1
	footer : 1
		divider : 1
		heading : 4
		nav-menu : 1
		search-form : 1
		social-icons : 1
		text-editor : 4
	header : 1
		icon-list : 1
		nav-menu : 1
		theme-site-logo : 1
	page : 2
		heading : 4
		spacer : 1
		theme-site-logo : 1
	section : 1
		heading : 1
		posts : 1
	single : 2
		global : 1
		heading : 1
		image : 1
		post-info : 1
		post-navigation : 1
		share-buttons : 1
		text-editor : 1
		theme-post-content : 2
		theme-post-title : 2
	widget : 0
		form : 2
	wp-post : 8
		accordion : 1
		alert : 4
		button : 17
		code-highlight : 2
		counter : 2
		divider : 2
		form : 1
		global : 5
		heading : 51
		icon-box : 7
		icon-list : 6
		image : 1
		lottie : 4
		portfolio : 1
		social-icons : 3
		table-of-contents : 1
		tabs : 3
		testimonial-carousel : 1
		text-editor : 90
		toggle : 1
	wp-page : 13
		accordion : 1
		alert : 1
		button : 13
		call-to-action : 1
		divider : 2
		facebook-page : 1
		global : 1
		heading : 19
		lottie : 1
		menu-anchor : 4
		posts : 3
		price-list : 2
		price-table : 12
		social-icons : 2
		text-editor : 26
		wp-widget-tag_cloud : 1
	single-post : 1
		button : 1
		divider : 1
		heading : 1
		image : 1
		portfolio : 1
		post-info : 1
		text-editor : 1
		theme-post-content : 1
		theme-post-title : 1
	single-page : 1
		shortcode : 1
		table-of-contents : 1
		text-editor : 1
		theme-post-content : 1
		theme-post-title : 1
	search-results : 1
		archive-posts : 1
		theme-archive-title : 1
	error-404 : 1
		heading : 2
		nav-menu : 1
		search-form : 1


== Log ==
	
Log: showing 20 of 422021-06-03 08:12:17 [info] Elementor/Upgrades - _on_each_version Start  
2021-06-03 08:12:18 [info] Elementor/Upgrades - _on_each_version Finished 
2021-06-03 08:12:18 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.3.0-dev11',
  'to' => '3.3.0-dev12',
)]
2021-06-03 08:12:18 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.3.0-dev11',
  'to' => '3.3.0-dev12',
)]
2021-06-04 16:56:34 [info] elementor::elementor_updater Started 
2021-06-04 16:56:34 [info] Elementor/Upgrades - _on_each_version Start  
2021-06-04 16:56:35 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.3.0-dev12',
  'to' => '3.3.0-dev13',
)]
2021-06-04 16:56:35 [info] Elementor/Upgrades - _on_each_version Finished 
2021-06-04 16:56:35 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.3.0-dev12',
  'to' => '3.3.0-dev13',
)]
2021-06-07 23:13:14 [info] elementor::elementor_updater Started 
2021-06-07 23:13:14 [info] Elementor/Upgrades - _on_each_version Start  
2021-06-07 23:13:14 [info] Elementor/Upgrades - _on_each_version Finished 
2021-06-07 23:13:14 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.3.0-dev13',
  'to' => '3.3.0-dev14',
)]
2021-06-07 23:13:14 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.3.0-dev13',
  'to' => '3.3.0-dev14',
)]
2021-06-08 18:26:08 [info] elementor::elementor_updater Started 
2021-06-08 18:26:08 [info] Elementor/Upgrades - _on_each_version Start  
2021-06-08 18:26:08 [info] Elementor/Upgrades - _on_each_version Finished 
2021-06-08 18:26:08 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.3.0-dev14',
  'to' => '3.3.0-dev15',
)]
2021-06-08 18:26:08 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.3.0-dev14',
  'to' => '3.3.0-dev15',
)]
2021-06-08 19:42:46 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.4.0-dev11',
  'to' => '3.4.0-dev13',
)]

PHP: showing 1 of 1PHP: 2021-06-04 06:00:27 [notice X 6][/app/data/public/wp-content/plugins/elementor-pro/license/updater.php::75] Undefined index: new_version [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
```
</details>

**Comment by shilo-ey** [comment]
Great feedback, thank you!

**Comment by d19dotca** [comment]
Ah I see you guys were already on top of that one, haha, excellent! 😄

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v3.3.0

Feel free to update

Thanks!

**Comment by gmariani** [comment]
Can this be disabled and/or removed? We already have a widget that gets added to every dashboard with just ads ( "news" ) from elementor. Now i have a widget at the top I have to explain to clients? One or the other, not both. Just replace the blog feed with a button to the account and you now have the best of both worlds. 

I do not care for this new widget, it's just more clutter/junk.
